code/modules/clothing/spacesuits/void/wizard.dm

```diff
@@ -19,17 +19,26 @@
 		var/obj/item/organ/external/LH = H.get_organ("l_hand")
 		var/obj/item/organ/external/RH = H.get_organ("r_hand")
 		var/active_hand = H.hand
-		user << "<span class='warning'>Your hand passes through the [src] with a flash of searing heat!</span>"
+		user << "<span class='warning'>As you pick it up, \the [src] flashes, and you feel a searing pain through your hand! You should put this down as soon as you can!</span>"
 		playsound(user, 'sound/effects/sparks4.ogg', 40, 1)
-		user.drop_item()
 		if(active_hand)
-			LH.droplimb(0,DROPLIMB_BURN)
+			LH.take_damage(15)
```

Comment:
15 damage seems pretty low. Especially considering it gibbed beforehand.

You could already move it by dragging it, so I don't see why this is an issue.

Reply:
It´s just a space suit that cant really be used to do anything with.
Therefore I dont see much issue with that.

Reply:
I've watched it burn off people's hands by dragging it before.

And it doing that is why I changed it.